Xamarin.Forms.Platform.UAP/DatePickerRenderer.cs

```diff
@@ -50,6 +56,20 @@ protected override void OnElementChanged(ElementChangedEventArgs<DatePicker> e)
 			base.OnElementChanged(e);
 		}
 
+		protected override void UpdateTabStop()
+		{
+			base.UpdateTabStop();
+			Control?.GetChildren<Control>().ForEach(c => c.IsTabStop = Element.IsTabStop);
+		}
+
+		void OnGettingFocus(UIElement sender, GettingFocusEventArgs args) => focusDirection = args.Direction;
+
+		void OnGotFocus(object sender, RoutedEventArgs e)
```

Comment:
This logic gets repeated multiple times throughout this PR. Maybe we could add a centralized place it gets handled, a static class or something?